lambda-notes/src/components/SearchBar.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+class SearchBar extends React.Component {
+
+handleSubmit = event => {
+  event.preventDefault();
+}
+  render(){
+    return (
+      <div className='search-bar'>
+      <form onSubmit={this.handleSubmit}>
+        <input
+          type='text'
+          name='searchInput'
+          defaultValue=''
+          placeholder='search notes'
+          onChange={this.props.handleInput}
+        />
+        </form>
+      </div>
+    )
+  }
+}
+
+const mapStateToProps = state => {
+  return {
+    notes: state.notesReducer.notes,
+  }
+}
+
+export default connect(
```

Comment:
```suggestion
export default connect(mapStateToProps)(SearchBar)
```

If you are not making use of mappingDispatchToProps, you do not have to give it an empty object. Simply do not pass it anything, not even an empty object. Doing this will provide the dispatch method to the current component. dispatch will become accessible as a props if you don’t provide a mapDispatchToProps.